client/src/App.js

```diff
@@ -22,7 +23,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList}></Route>
+        <Route exact path="/movies/:id" component={Movie}></Route>
```

Comment:
you can omit the exact from the path='/movies/:id' Route, its only needed on the '/' Route in this instance